src/components/table/Table.vue

```diff
@@ -86,8 +86,23 @@
                                     />
                                 </template>
                                 <template v-else>{{ column.label }}</template>
-
+                                <template
+                                    v-if="backendSorting &&
+                                        sortMultiple &&
+                                        sortMultipleData.find(i =>
+                                    i.field === column.field)">
+                                    <b-icon
+                                        :icon="sortIcon"
+                                        :pack="iconPack"
+                                        both
+                                        :size="sortIconSize"
+                                        :class="{ 'is-desc': isDesc(column)}"
+                                    />
+                                    {{ sortMultipleData.findIndex(i =>
```

Comment:
Please avoid to add logic in the template, you might use a computed prop

Reply:
The logic isDesc to find the current row in the data i pass. Not sure how i would fix that one, if thats what you mean. 
I changed line 101 to pass whatever you like, ex a,b,c instead of 1,2,3